### ISSUE TITLE
[Request] Client and Server Manager support ECDSA certificates

### ISSUE THREAD

**Issue by 0F4mpzDafcLwFugk** [SoftEtherVPN/SoftEtherVPN#1034]
### Prerequisites
**SoftEther version: 5.01.9671**
built from source code https://github.com/SoftEtherVPN/SoftEtherVPN.git
**Operating system: Windows**
### Description
If I import ECDSA cert, the Manager will throw error.
![client](https://user-images.githubusercontent.com/22396764/69014191-0f6c7a80-09cb-11ea-8aa3-1691be394799.png)
![server](https://user-images.githubusercontent.com/22396764/69014193-11ced480-09cb-11ea-8986-74ed0f6b351e.png)
<br>
**_But when I manually put cert and key into vpn_server.config on server, it works fine._**
<img width="1021" alt="cert key" src="https://user-images.githubusercontent.com/22396764/69013926-7e949f80-09c8-11ea-8496-8a801b089653.png">
My vpnserver's SSL Labs test. It supports ECDSA unquestionably.
### server removed
<img width="710" alt="test" src="https://user-images.githubusercontent.com/22396764/69014059-bbad6180-09c9-11ea-960a-d76ab4c26c94.png">
### In conclusion
The vpnserver and vpnclient support ECDSA certificates. But Manager GUI doesn't.
